pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -48,3 +77,41 @@ func (pc *PackageControllerClient) ValidateControllerDoesNotExist(ctx context.Co
 	}
 	return nil
 }
+
+func (pc *PackageControllerClient) ApplySecret(ctx context.Context) error {
+	eksaAccessKeyId, eksaSecretAccessKey := os.Getenv(config.EksaAccessKeyIdEnv), os.Getenv(config.EksaSecretAcessKeyEnv)
+	eksaRegion := eksaDefaultRegion
+	if region, found := os.LookupEnv(config.EksaRegionEnv); found {
+		eksaRegion = region
+	}
+
+	templateValues := map[string]string{
+		"eksaAccessKeyId":     base64.StdEncoding.EncodeToString([]byte(eksaAccessKeyId)),
```

Comment:
If you save it in the secret as `stringData`, you may not need to base64 encode it here but I think this is fine.

Reply:
Is this a data type? I wasn't able to find stringData.

Reply:
yea its a field in the yaml: https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret

Reply:
aaah I see. Let me try that.